docs/Antora/modules/knowledge_base/pages/protocol_features_characteristics.adoc

```diff
@@ -0,0 +1,351 @@
+= Protocol features and characteristics
+
+This article features an overview of the different features of each server/client protocol currently supported in the main openDAQ(TM) repository. We aim to outline different capabilities and features made available by the SDK's core, as well as indicate which of the protocols support said feature. 
+
+Additionally, we provide metrics for protocol compatibility within a given openDAQ(TM) system of devices, detailing which servers/clients can be used in conjunction with different system setups.
+
+== Brief Server/Client architecture overview
+
+The SDK is structured in a manner where each device represents its structure in the form of a tree of components, with each component having a set of properties. We use a server-client architecture to forward the information about the device's tree structure from server to client. A device will start a server, advertising its capabilities over a discovery server, allowing clients to find the device, and connect to it via a corresponding client implementation. In general, server-client connections in openDAQ(TM) are split into two different types:
+
+* **Configuration**: The server sends information on its tree structure and properties to the client, allowing the client to produce a mirror of the device's component tree. This allows the client-side application to act as if the device was local to the client instance, traversing its tree, and modifying its property values. A configuration connection does not transfer signal data.
+* **Streaming**: The server sends information on what signals it has available to the client, and sends the data sent through the signals if the client is subscribed to said signals. This form of connection is usually combined with a configuration connection, allowing users to see the whole device structure, as well as read the device's signal data. If only a streaming connection is formed, the client will not see the full component tree, but will still be able to see a list of the device's signals directly underneath the device's "signals" folder.
```

Comment:
The sentence seems a bit confusing. I would rephrase it if possible, e.g.: `The server sends to the client: the information on what signals it has available; and the data sent through the signals if the client is subscribed to said signals.`